R/filter_by_feature.R

```diff
@@ -2,69 +2,65 @@
 #'
 #' This function filters a matrix based on specified features and conditions.
 #'
-#' @param fltr_features A character vector specifying the features to filter.
-#' @param fltr_condition A character vector specifying the conditions for each feature.
+#' @param filter_features A character vector specifying the features to filter.
+#' @param filter_conditions A character vector specifying the conditions for each feature.
 #' @param matrix The input matrix to filter.
 #' @param verbose Logical indicating whether to display verbose output (default is TRUE).
 #'
 #' @return A list containing the filtered matrix and the indices of the filtered rows.
 #'
 #' @examples
 #' filter_by_feature(
-#'   fltr_features = c("landpercentage", "forestmask"),
-#'   fltr_condition = c(">100", ">0"),
+#'   filter_features = c("landpercentage", "forestmask"),
```

Comment:
not sure but if these are happen to change, it would be better to have them in the config file